gcc/d/gdc.texi

```diff
@@ -120,196 +120,662 @@ and the Info entries for @file{gdc} and @file{gcc}.
+@item -funittest
+@cindex @option{-funittest}
+@cindex @option{-fno-unittest}
+Turns on compilation of @code{unittest} code, and turns on the unittest
+version identifier.  This implies @option{-fassert}.
 
-These are supported options:
+@item -fversion=@var{value}
+@cindex @option{-fversion}
+Turns on compilation of conditional @code{version} code into the program
+identified by any of the following values:
 
 @table @samp
 @item level
-Compile in debug code <= level.
-
+@cindex @option{-fversion=level}
+Sets the version level to @var{level}, any @code{version} code >= @var{level}
+is compiled into the program.
 @item ident
-Compile in debug code identified by ident.
+@cindex @option{-fversion=ident}
+Turns on compilation of @code{version} code identified by @var{ident}.
 @end table
 
-@item -fdeps=@var{filename}
-@cindex @option{-fdeps}
-Write module dependencies to filename.
+@end table
 
-@item -fmake-deps=@var{filename}
-@cindex @option{-fmake-deps}
-Write makefile dependency output to the given file.
+@node Directory Options
+@section Options for Directory Search
+@cindex directory options
+@cindex options, directory search
+@cindex search path
 
-@item -fmake-mdeps=@var{filename}
-@cindex @option{-fmake-mdeps}
-Like -fmake-deps=@var{filename} but ignore system header files.
+These options specify directories to search for files, libraries, and
+other parts of the compiler:
 
-@item -fonly=@var{filename}
-@cindex @option{-fonly}
-Process all modules specified on the command line,
-but only generate code for the module specified by the argument.
+@table @gcctabopt
 
-@item -fversion=@var{opt}
-@cindex @option{-fversion}
-Compile in version code into the program.
+@item -I@var{dir}
+@cindex @option{-I}
+Specify a directory to use when searching for imported modules at
+compile time.  Multiple @option{-I} options can be used, and the
+paths are search in the same order.
+
+@item -J@var{dir}
+@cindex @option{-J}
+Specify a directory to use when searching for files in string imports
+at compile time.  This switch is required in order to use
+@code{import(file)} expressions.  Multiple @option{-J} options can be
+used, and the paths are searched in the same order.
+
+@item -L@var{dir}
+@cindex @option{-L}
+When linking, specify a library search directory, as with @command{gcc}.
+
+@item -B@var{dir}
+@cindex @option{-B}
+This option specifies where to find the executables, libraries,
+source files, and data files of the compiler itself.
+
+@item -fmodule-filepath=@var{module}=@var{spec}
+@cindex @option{-fmodule-filepath}
+This option manipulates file paths of imported modules, such that if an
+imported module matches all or the leftmost part of @var{module}, the file
+path in @var{spec} is used as the location to search for D sources.
+This is used when the source file path and names are not the same as the
+package and module hierachy.
+
+For example, compiling with the following @option{-fmodule-filepath}
+options.
```

Comment:
I'd prefer somehow marking that this sentence continues, e.g. use `...`.

Or maybe better rewrite so that we have two independent sentences e.g.

> Consider the following examples:
>...
> This will tell the compiler to search....

This is especially annoying as there's a page break after the first sentence in the PDF ;-)